index.html

```diff
@@ -5,79 +5,108 @@
     <meta name="website" content="NF website Hugh" />
     <title>My First NF Website</title>
 
-    <!-- <link href="./style.css" rel="stylesheet" />
-    -->
+    <link href="./style.css" rel="stylesheet" />
   </head>
 
   <body>
     <header id="Home">
       <h1>Hugh Burgess</h1>
       <nav>
         <ul>
-          <a href="#Home"><li>Home</li></a>
-          <a href="#Contact"><li>Contact</li></a>
-          <a href="#Career"><li>Career</li></a>
+          <li>
+            <a href="#Home">Home 🏡</a>
+          </li>
+          <li>
+            <a href="#Work">Work 🗄</a>
+          </li>
+          <li>
+            <a href="#AboutMe">About Me 🤖</a>
+          </li>
+          <li>
+            <a href="#Contact">Contact 📞</a>
+          </li>
+
+          <li>
+            <a href="#CV">CV 📨</a>
+          </li>
         </ul>
       </nav>
 
       <br />
 
       <img src="/images/profile-picture.png" alt="Picture of Hugh" />
     </header>
-    <section>
-      <h2 id="Career">About Me</h2>
-      <article>
-        <p>
-          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quo, saepe
-          aperiam alias vitae quod autem iusto untur itaque in vel quidem dicta
-          temporibus cupiditate voluptatum numquam velit perferendis maiores
-          animi!
-        </p>
-      </article>
-    </section>
-
-    <section>
-      <div class="contact_box">
-        <p>Mobile: 07234823492342213434</p>
-        <p>Email: <a href="">hello@me.com</a></p>
-      </div>
-    </section>
 
-    <article class="links" id="Contact">
-      <h1>Contact</h1>
+    <article class="links" id="Work">
+      <h1>Work</h1>
       <hr />
 
-      <img src="./images/github.png" alt="Photo of GitHub Logo" />
-      <a class="hyperlink_text" href="https://github.com/">GitHub</a>
+      <a href="https://github.com/hugh-burgess"
+        ><img
+          class="github"
+          src="./images/github.png"
+          alt="Photo of GitHub Logo"
+      /></a>
+      <a class="hyperlink_text" href="https://github.com/hugh-burgess"
+        >GitHub</a
+      >
       <hr />
-      <img src="./images/codepen.png" alt="Photo of CodePen" />
-      <a class="hyperlink_text" href="https://codepen.com/">CodePen</a>
+      <a href="https://codepen.io/hugh-burgess"
+        ><img class="codepen" src="./images/codepen.png" alt="Photo of CodePen"
+      /></a>
+      <a class="hyperlink_text" href="https://codepen.io/hugh-burgess"
+        >CodePen</a
+      >
       <hr />
-      <img src="./images/linkedin.png" alt="Photo of LinkedIn" />
-      <a class="hyperlink_text" href="https://linkedin.com/">LinkedIn</a>
+      <a href="https://www.linkedin.com/in/hugh-burgess"
+        ><img
+          class="linkedin"
+          src="./images/linkedin.png"
+          alt="Photo of LinkedIn"
+      /></a>
+      <a class="hyperlink_text" href="https://www.linkedin.com/in/hugh-burgess"
+        >LinkedIn</a
+      >
```

Comment:
Here you could use a list for your links.

Reply:
that's interesting, but wouldnt that totally change my layout?

Reply:
was only a though. will check.

Reply:
I know what you mean. I would try it. A list is the better symantic. If it changes your layout, don't worry. Next week you will learn how to style it back the way you want it.